src/CMakeLists.txt

```diff
@@ -203,7 +205,8 @@ target_link_libraries(keepassx_core
                       Qt5::Widgets
                       ${GCRYPT_LIBRARIES}
                       ${GPGERROR_LIBRARIES}
-                      ${ZLIB_LIBRARIES})
+                      ${ZLIB_LIBRARIES}
+                      ${OATH_LIBRARIES})
```

Comment:
Are the libraries readily available on both MinGW and OSX? If so I say leave it as-is and update the wiki. This should be treated like core capability and not a plugin.